mesa_geo/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 __all__ = ["GeoSpace", "GeoAgent", "AgentCreator"]
 
-__title__ = "mesa-geo"
-__version__ = "0.1.2"
+__title__ = "Mesa-Geo"
+__version__ = "0.2.0"
```

Comment:
Just noticed this. We should follow [semantic versioning](https://semver.org/). Reviewing all the new commits so far, they are bug fixes, no new functionalities. So it should be "0.1.3".

Reply:
Actually I didn't increase the version... The most recent version number for mesa-geo is `0.2.0`: https://github.com/projectmesa/mesa-geo/tags and https://pypi.org/project/mesa-geo/

Btw for the next release we might need to prefix it with the letter "v" (e.g. `v0.2.1`) to be consistent with Mesa. This letter was there for previous releases of mesa-geo, but somehow was dropped for `0.2.0`.

Perhaps we can bump it to `v0.2.1` when we get the access to pypi.

Reply:
OK sounds good.